### PR TITLE
ISSUE-70 Expose NODE_TIER as Env Var

### PR DESCRIPTION
**Pull request by MadhuriArugula**
Exposing NODE_TIER as environment variable as per issue #70

### CODE REVIEW

**Comment by MadhuriArugula** [comment]
> you should change to
> Specify a node name to classify the tiers.

Updated as per the comment. Thanks.

**Comment by dcasavant** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Name 						| Purpose 	| Default
 NODE_TYPE 					| Specify a node type or classification to specialize the processing within this container.  for more information, see  [Node types for on-premises environments](https://community.pega.com/sites/default/files/help_v83/procomhelpmain.htm#engine/node-classification/eng-node-types-ref.htm). |
 PEGA_DIAGNOSTIC_USER 		| Set a Pega diagnostic username to download log files. |
 PEGA_DIAGNOSTIC_PASSWORD 	| Set a secure Pega diagnostic username to download log files. |
-
+NODE_TIER                   | Specify node name to classify the tiers. |
```
[the remark this answers is not in the document]
Comment (MadhuriArugula) [reply README.md]: @dcasavant This looks more precise to me with a minor edit - "Specify the display name of the tier that this node is a logical part of" @taz-mon

[the remark this answers is not in the document]
Comment (taz-mon) [reply README.md]: Since we want more precision, can we say "Specify the display name of the tier to which you logically associate this node."

**Pull request merged**